File: online-judge-solutions/Codeforces/1552A.py
#https://codeforces.com/problemset/problem/1552/A
#Lang		:	Python 3.8
#Time		:	46 ms
#Memory         :	0 KB
t=int(input())
for x in range(0,t):
    n,s,p=int(input()),input(),0
    r=s
    s=''.join(sorted(s))
    for i in range(n):
        if(s[i]!=r[i]):p+=1
    print(p)

File: online-judge-solutions/Codeforces/1552A.cpp
//https://codeforces.com/problemset/problem/1552/A
//Lang		:	GNU C++14
//Time		:	15 ms
//Memory	:	0 KB
#include<iostream>
#include<algorithm>
using namespace std;
int main(){
	int t,n;
	cin>>t;
	while(t--){
		string s;
		cin>>n>>s;
		string r=s;
		sort(s.begin(),s.end());
		int p=0;
		for(int i=0;i<s.size();i++){
			if(s[i]!=r[i]) p++;
		}
		cout<<p<<endl;
	}
	return 0;
}
